plugins/node/opentelemetry-instrumentation-aws-sdk/test/sqs.test.ts

```diff
@@ -361,6 +363,49 @@ describe('SQS', () => {
   });
 
   describe('hooks', () => {
+    it('sqsResponseHook add messaging attributes', async () => {
+      const region = 'us-east-1';
+      const sqs = new AWS.SQS();
+      sqs.config.update({ region });
+
+      const QueueName = 'unittest';
+      const params = {
+        QueueUrl: `queue/url/for/${QueueName}`,
+        MessageBody: 'payload example from v2 without batch',
+      };
+
+      const response = await sqs.sendMessage(params).promise();
+
+      expect(getTestSpans().length).toBe(1);
+      const [span] = getTestSpans();
+
+      // make sure we have the general aws attributes:
+      expect(span.attributes[SemanticAttributes.RPC_SYSTEM]).toEqual('aws-api');
+      expect(span.attributes[SemanticAttributes.RPC_METHOD]).toEqual(
+        'SendMessage'
+      );
+      expect(span.attributes[SemanticAttributes.RPC_SERVICE]).toEqual('SQS');
+      expect(span.attributes[AttributeNames.AWS_REGION]).toEqual(region);
+
+      // custom messaging attributes
+      expect(span.attributes[SemanticAttributes.MESSAGING_SYSTEM]).toEqual(
+        'aws.sqs'
+      );
+      expect(
+        span.attributes[SemanticAttributes.MESSAGING_DESTINATION_KIND]
+      ).toEqual(MessagingDestinationKindValues.QUEUE);
+      expect(span.attributes[SemanticAttributes.MESSAGING_DESTINATION]).toEqual(
+        QueueName
+      );
+      expect(span.attributes[SemanticAttributes.MESSAGING_URL]).toEqual(
+        params.QueueUrl
+      );
+      expect(span.attributes[SemanticAttributes.MESSAGING_MESSAGE_ID]).toEqual(
+        response.MessageId
+      );
+      // expect(span.attributes[SemanticAttributes.HTTP_STATUS_CODE]).toEqual(200);
```

Comment:
It seems to be `undefined` with aws sdk v2 and `200` with aws sdk v3. Haven't researched why.

Reply:
Might be related to the mock response returned for these tests [here](https://github.com/open-telemetry/opentelemetry-js-contrib/blob/c5b9356d4eb925df64b66c859d1d8367ae57d437/plugins/node/opentelemetry-instrumentation-aws-sdk/test/testing-utils.ts#L52).

It's ok to merge as is, but if you have the motivation you are very welcome to see if it's an easy fix and help improve the tests :)

Reply:
You were right. Lack of types makes it hard to contribute to the repository :( Fixed in https://github.com/open-telemetry/opentelemetry-js-contrib/pull/968/commits/866e00f27ae50d78f5d9caf7e442c6b1dbe22e47.

Reply:
> Lack of types makes it hard to contribute to the repository

I agree. If you have some free time and want to contribute by adding them, that would be great :)